models/staging/raw_payments.py

```diff
@@ -1,6 +1,19 @@
 import pandas as pd
+from sshfs import SSHFileSystem
+
 
 def model(dbt, session) -> pd.DataFrame:
     dbt.config(materialized="table")
-    df = pd.read_json("http://35.196.10.176:8000/payments", orient="records")
+    dbt.config(fal_environment="sftp")
+
+    hostname = "35.193.199.9"
```

Comment:
Can we centralize this information in a [model's config?](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/python-models#accessing-project-context)

Reply:
Thanks for the comments, @chamini2. I decided to use env vars. Let me know what you think.

Reply:
I see that it works, my concern is that dbt seems to recommend even accessing env vars with `dbt.config.get`

https://docs.getdbt.com/docs/building-a-dbt-project/building-models/python-models#accessing-project-context

```yaml
version: 2

models:
  - name: my_python_model
    config:
      materialized: table
      target_name: "{{ target.name }}"
      specific_var: "{{ var('SPECIFIC_VAR') }}"
      specific_env_var: "{{ env_var('SPECIFIC_ENV_VAR') }}"
```

```py
def model(dbt, session):
    target_name = dbt.config.get("target_name")
    specific_var = dbt.config.get("specific_var")
    specific_env_var = dbt.config.get("specific_env_var")

    orders_df = dbt.ref("fct_orders")

    # limit data in dev
    if target_name == "dev":
        orders_df = orders_df.limit(500)
```

We can leave it as is, just wanted to point it out. It could be a good idea to follow dbt's guidelines.

Reply:
Actually, we should change it to what dbt suggests. If we change configurations to run this script on a different server (e.g. dataproc), the environment won't be available for the Python script.